packages/shorebird_cli/lib/src/commands/patch/patch_android_command.dart

```diff
@@ -116,18 +116,9 @@ If this option is not provided, the version number will be determined from the p
     final app = await codePushClientWrapper.getApp(appId: appId);
 
     var hasBuiltWithActiveFlutter = false;
+    late File aabFile;
```

Comment:
Does this need to be `late`?

Reply:
Yes, since it is initialize down in the code. The alternative would be to make it nullable, but I feel that would make the code more cumbersome.

Reply:
Yep, fair enough!